spacy/tokens/token.pyx

```diff
@@ -633,6 +633,18 @@ cdef class Token:
             yield self.doc[head_ptr - (self.c - self.i)]
             i += 1
 
+    @property
+    def root(self):
```

Comment:
This property on the `Token` does something different from the `span.root`, so I wouldn't want it to be part of the public API.

We should just move the two lines here into the body of `is_ancestor`.